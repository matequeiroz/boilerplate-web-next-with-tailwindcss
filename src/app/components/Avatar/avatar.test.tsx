import { render, screen } from '@testing-library/react'

import { Avatar } from '.'

describe('<Avatar />', () => {
  it('should render the avatar component', () => {
    // renderiza o component
    const { container } = render(<Avatar />)

    // busca o elemento e verifica a existÃªncia dele
    expect(screen.getByRole('img')).toBeInTheDocument()

    // gerar snapshot
    expect(container.firstChild).toMatchSnapshot()
  })
})
