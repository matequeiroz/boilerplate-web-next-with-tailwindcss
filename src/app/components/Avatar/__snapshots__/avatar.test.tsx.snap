// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar /> should render the avatar component 1`] = `
<div>
  <img
    alt="avatar"
    class="w-full"
    data-nimg="1"
    decoding="async"
    height="300"
    loading="lazy"
    src="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1486693326701-1ea88c6e2af3%3Fixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D%26auto%3Dformat%26fit%3Dcrop%26w%3D2787%26q%3D80&w=640&q=75"
    srcset="/_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1486693326701-1ea88c6e2af3%3Fixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D%26auto%3Dformat%26fit%3Dcrop%26w%3D2787%26q%3D80&w=384&q=75 1x, /_next/image?url=https%3A%2F%2Fimages.unsplash.com%2Fphoto-1486693326701-1ea88c6e2af3%3Fixlib%3Drb-4.0.3%26ixid%3DM3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%253D%253D%26auto%3Dformat%26fit%3Dcrop%26w%3D2787%26q%3D80&w=640&q=75 2x"
    style="color: transparent;"
    width="300"
  />
</div>
`;
